### ISSUE TITLE
Tooltip-template is not valid HTML

### ISSUE THREAD

**Issue by ArthurWietzorek** [clientIO/joint#1310]
I am using Rappid 3.1.1 in an Angular application that i recently upgraded to Angular 9. After the update, all tooltips created with `joint.ui.Tooltip` were broken. The following picture shows a tooltip on a halo handle.

![image](https://user-images.githubusercontent.com/41048767/82909084-1ae7c200-9f69-11ea-90d5-670a5abd0a0d.png)

The HTML, which previously looked like this:
![image](https://user-images.githubusercontent.com/41048767/82910328-d2310880-9f6a-11ea-82ac-212033c9d4a8.png)
... suddenly had the tooltip elements nested inside of each other:
![image](https://user-images.githubusercontent.com/41048767/82910551-0b697880-9f6b-11ea-9392-34bd997c32b1.png)

After searching for a while, i found the following lines in the rappid.js file (lines 54734-54735):
```
/** @type {string} */
template: '<div class="tooltip-arrow"/><div class="tooltip-arrow-mask"/><div class="tooltip-content"/>',
```
... which is not valid HTML, as `<div>` is not a void element and therefore does not support the self-closing tag (see https://www.w3.org/TR/html5/syntax.html#start-tags). I am not sure why this problem did not occur when using Rappid with earlier Angular versions, but changing the mentioned lines to:
```
/** @type {string} */
template: '<div class="tooltip-arrow"></div><div class="tooltip-arrow-mask"></div><div class="tooltip-content"></div>',
```
... solves the problem.
![image](https://user-images.githubusercontent.com/41048767/82909999-53d46680-9f6a-11ea-83b9-82b347bc1f32.png)

**Comment by kumilingus** [comment]
Thank you @ArthurWietzorek. We've already come across this issue and fixed it for the next release. It is odd. This XML template string is parsed by jQuery internally. Perhaps the jQuery version changed throughout the upgrade in your case? We also have an application using Angular 9 here without this issue.

You can also pass a new template to the Tooltip constructor (instead of overriding the library).

```
new joint.ui.Tooltip({
  template: '<div class="tooltip-arrow"></div><div class="tooltip-arrow-mask"></div><div class="tooltip-content"></div>'
});
```

**Comment by kumilingus** [comment]
Fixed in `v3.2.0`.
